greenery/fsm.py

```diff
@@ -72,7 +72,7 @@ def __post_init__(self):
                 f"Final states {repr(self.finals)} "
                 f"must be a subset of {repr(self.states)}"
             )
-        for state, symbol in self.map.items():
+        for state, _state_trans in self.map.items():
```

Comment:
Does this change not break the rest of the code?

Reply:
It does not, because the name `symbol` was re-used two lines below:
```py
for symbol in self.map[state]:
```

This binding was never used.